lib/components/app_drawer/app_drawer.dart

```diff
@@ -119,73 +119,97 @@ class AppDrawer extends StatelessWidget {
                     ),
                   ),
                   Container(
-                      padding: const EdgeInsets.all(21),
-                      alignment: Alignment.centerLeft,
-                      child: Row(
-                        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                        children: [
-                          Column(
-                            crossAxisAlignment: CrossAxisAlignment.start,
-                            children: [
-                              Padding(
-                                padding: const EdgeInsets.all(8.0),
-                                child: FloatingActionButton(
-                                  elevation: 0,
-                                  tooltip: appLocalizationsOf(context).help,
-                                  onPressed: () => launch(R.helpLink),
-                                  child: const Icon(Icons.help_outline),
-                                ),
-                              ),
-                              FutureBuilder(
-                                future: PackageInfo.fromPlatform(),
-                                builder: (BuildContext context,
-                                    AsyncSnapshot<PackageInfo> snapshot) {
-                                  if (snapshot.hasData) {
-                                    return Padding(
-                                      padding: const EdgeInsets.all(8.0),
-                                      child: Text(
-                                        appLocalizationsOf(context)
-                                            .appVersion(snapshot.data!.version),
-                                        style: Theme.of(context)
-                                            .textTheme
-                                            .caption!
-                                            .copyWith(color: Colors.grey),
-                                      ),
-                                    );
-                                  } else {
-                                    return const SizedBox(
-                                        height: 32, width: 32);
-                                  }
-                                },
+                    padding: const EdgeInsets.all(21 + 8),
```

Comment:
Consider using symmetric padding here to avoid adding that extra 2x8px vertically
```suggestion
                    padding: const EdgeInsets.symmetric(
                      horizontal: 21 + 8,
                      vertical: 21,
                    ),
```